k8s/client.go

```diff
@@ -638,11 +639,26 @@ func (client *Client) DeployCoreOperatorSync(ctx context.Context, coreCloudURL,
 		ImagePullPolicy: apiv1.PullAlways,
 		Env:             env,
 	}
+
+	resources := corev1.ResourceRequirements{
+		Limits: corev1.ResourceList{},
+	}
+
+	ml, err := resource.ParseQuantity(memoryLimit)
+	if err == nil {
+		resources.Limits = corev1.ResourceList{
+			"memory": ml,
+		}
+	}
```

Comment:
We should set a default in case this doesn't parses or comes incorrect.

Reply:
The default value is set to 512Mi

Reply:
We didn't have resources before. So my assumption is that it is determined on the cluster level.
I will have to look this up